src/xrCore/LocatorAPI.cpp

```diff
@@ -632,6 +631,7 @@ bool ignore_path(const char* _path)
     else
         return true;
 #elif defined(LINUX)
+    while (char* sep = strchr((char *)_path, '\\')) *sep = '/';
```

Comment:
Please consider moving this code (slashes conversion) to separate function (named like ConvertGamePathToPlatformSpecificPath) with platform-specific implementation. It makes the code more clear and allows to reduce code duplication.